tests/MEDS_Transforms/test_tokenization.py

```diff
@@ -237,3 +299,23 @@ def test_tokenization():
         want_data={**WANT_SCHEMAS, **WANT_EVENT_SEQS},
         should_error=True,
     )
+
+
+def test_tokenization_missing_static():
+    single_stage_transform_tester(
```

Comment:
I think you can just put this in the `test_tokenization()` function, maybe? I'm not sure it really needs a separate function. but this doesn't really matter much either way.